pkg/handlers/clusterlogging/handler.go

```diff
@@ -37,38 +40,66 @@ func NewHandlers(opts *config.Options) []handlers.RequestHandler {
 }
 
 func (ext *handler) Process(req *http.Request, context *handlers.RequestContext) (*http.Request, error) {
-	name := context.UserName
-	if context.IsWhiteListed(name) || ext.hasInfraRole(context) {
-		log.Debugf("Skipping additional processing, %s is whitelisted or has the infra role", name)
-		return req, nil
+
+	log.Infof("Received context: UserName: %v", context.UserName)
+	log.Infof("Received context: Roles: %v", context.Roles)
+
+	// if we are getting a .kibana/config url...
+	if ext.isKibanaIndex(*req.URL) {
+		kibanaVersion := ext.getKibanaVersion(*req.URL)
+		if kibanaVersion != "" {
+			log.Infof("Received request: kibana ver: %v", kibanaVersion)
```

Comment:
Again...asking but not with any preference.... should this be debug?  Any reason to log this regardless if its empty?

Reply:
will be removed for final review. this was for me.